shuffle/components.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
+  "style": "default",
+  "rsc": true,
+  "tsx": true,
+  "tailwind": {
+    "config": "tailwind.config.js",
+    "css": "styles/tailwind.css",
+    "baseColor": "slate",
+    "cssVariables": true
+  },
+  "aliases": {
+    "components": "@/components",
+    "utils": "@/utils/style-utils"
```

Comment:
How does this work compared to ts path aliases? Whats the purpose of these?

Reply:
Its not just aliasing, it uses tailwind-merge and cva to compose components, but this js is not shipped to our users, because of ssr.

Reply:
Okay. But this it impact ts compilatiom? Or is this just for the CLI?